terraform/terraform_environment/ecs_admin.tf

```diff
@@ -59,8 +59,8 @@ resource "aws_ecs_task_definition" "admin" {
   family                   = "${local.environment}-admin"
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
-  cpu                      = 512
-  memory                   = 1024
+  cpu                      = 2048
```

Comment:
I don't think we need to change the values on `admin`, it'll never have more than 1 user.

Reply:
👍